src/condor_starter.V6.1/starter.cpp

```diff
@@ -2077,10 +2081,309 @@ Starter::jobEnvironmentCannotReady(int status, const struct UnreadyReason & urea
 		return 0;
 	}
 
+	// Ask the AP what to do.
+	std::ignore = daemonCore->Register_Timer(
+		0, (TimerHandlercpp) & Starter::requestGuidanceJobEnvironmentUnready,
+		"ask AP what to do",
+		this
```

Comment:
Can we use the `std::function` version of `Register_Timer` here?

Reply:
Yes.